{"message":"Something failed during startup.","stack":"Error: Something failed during startup.\n    at Object.<anonymous> (C:\\Users\\herrifransisca\\Documents\\code\\node-course\\vidly\\index.js:27:7)\n    at Module._compile (internal/modules/cjs/loader.js:688:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)\n    at Module.load (internal/modules/cjs/loader.js:598:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:529:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)\n    at startup (internal/bootstrap/node.js:285:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)","level":"error","timestamp":"2018-11-30T13:20:53.141Z"}
{"message":"Something failed miserably!","stack":"Error: Something failed miserably!\n    at Object.<anonymous> (C:\\Users\\herrifransisca\\Documents\\code\\node-course\\vidly\\index.js:34:26)\n    at Module._compile (internal/modules/cjs/loader.js:688:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)\n    at Module.load (internal/modules/cjs/loader.js:598:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:529:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)\n    at startup (internal/bootstrap/node.js:285:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)","level":"error","timestamp":"2018-12-01T15:27:21.293Z"}
{"level":"info","message":"Connected to MongoDB...","timestamp":"2018-12-02T12:45:23.799Z"}
{"message":"failed to reconnect after 30 attempts with interval 1000 ms","stack":"MongoNetworkError: failed to reconnect after 30 attempts with interval 1000 ms\n    at Connection.<anonymous> (C:\\Users\\herrifransisca\\Documents\\code\\node-course\\vidly\\node_modules\\mongodb-core\\lib\\connection\\pool.js:364:13)\n    at Connection.emit (events.js:182:13)\n    at Socket.<anonymous> (C:\\Users\\herrifransisca\\Documents\\code\\node-course\\vidly\\node_modules\\mongodb-core\\lib\\connection\\connection.js:246:50)\n    at Object.onceWrapper (events.js:273:13)\n    at Socket.emit (events.js:182:13)\n    at emitErrorNT (internal/streams/destroy.js:82:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)\n    at process._tickCallback (internal/process/next_tick.js:63:19)","name":"MongoNetworkError","errorLabels":["TransientTransactionError"],"level":"error","timestamp":"2018-12-02T12:47:23.286Z"}
{"level":"info","message":"Listening on port 3000...","timestamp":"2018-12-02T13:14:14.800Z"}
{"level":"info","message":"Connected to MongoDB...","timestamp":"2018-12-02T13:14:15.848Z"}
